package.json

```diff
@@ -28,41 +28,29 @@
   },
   "dependencies": {
     "rehype-react": "^6.0.0",
-    "remark-parse": "^8.0.0",
-    "remark-rehype": "^7.0.0",
+    "remark-parse": "^9.0.0",
+    "remark-rehype": "^8.0.0",
     "unified": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.4",
-    "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-docs": "^5.3.19",
-    "@storybook/addon-info": "^5.3.19",
-    "@storybook/addon-knobs": "^5.3.19",
-    "@storybook/addon-links": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
-    "@storybook/react": "^5.3.19",
-    "@testing-library/jest-dom": "^5.11.0",
-    "@testing-library/react": "^10.4.3",
-    "@testing-library/react-hooks": "^3.3.0",
-    "@types/react": "^16.9.41",
-    "@types/react-dom": "^16.9.8",
-    "babel-loader": "^8.1.0",
-    "husky": "^4.2.5",
-    "jest-environment-jsdom-sixteen": "^1.0.3",
-    "katex": "^0.11.1",
-    "react": "^16.13.1",
-    "react-docgen-typescript-loader": "^3.7.2",
-    "react-dom": "^16.13.1",
-    "react-is": "^16.13.1",
-    "react-test-renderer": "^16.13.1",
-    "rehype-katex": "^3.0.0",
-    "rehype-raw": "^4.0.2",
-    "rehype-sanitize": "^3.0.1",
-    "remark-math": "^2.0.1",
-    "ts-loader": "^7.0.5",
-    "tsdx": "^0.13.2",
-    "tslib": "^2.0.0",
-    "typescript": "^3.9.6"
+    "@babel/core": "^7.0.0",
+    "@storybook/addon-essentials": "^6.0.0",
+    "@storybook/react": "^6.0.0",
+    "@testing-library/jest-dom": "^5.0.0",
+    "@testing-library/react": "^11.0.0",
+    "@testing-library/react-hooks": "^3.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "husky": "^4.0.0",
+    "katex": "^0.12.0",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "rehype-katex": "^4.0.0",
+    "rehype-raw": "^5.0.0",
+    "rehype-sanitize": "^4.0.0",
+    "remark-math": "^4.0.0",
+    "tsdx": "^0.14.0",
+    "typescript": "^3.0.0"
```

Comment:
typescript 4 will come in a future update when it has official support in tsdx https://github.com/formium/tsdx/issues/926